oioioi/interactive/models.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from oioioi.filetracker.fields import FileField
+from oioioi.problems.models import Problem, make_problem_filename
+
+
+class Interactor(models.Model):
+    problem = models.OneToOneField(Problem, on_delete=models.CASCADE)
```

Comment:
Since this is a one-to-one model, perhaps it is better to use [model iheritance](https://docs.djangoproject.com/en/4.0/topics/db/models/#model-inheritance) here?